dfx.json

```diff
@@ -1,5 +1,5 @@
 {
-  "dfx": "0.14.2",
+  "dfx": "0.13.1",
```

Comment:
I think we should keep it on 0.14.2 but just for local testing you'll need to currently use 0.13.1.
When we deploy to prod we should still use latest.

Reply:
It's back!